pandas_gbq/auth.py

```diff
@@ -12,40 +12,34 @@
 logger = logging.getLogger(__name__)
 
 
+CREDENTIALS_CACHE_DIRNAME = "pandas_gbq"
+CREDENTIALS_CACHE_FILENAME = "bigquery_credentials.dat"
```

Comment:
To confirm, you think we should have something specific for `pandas_gbq` rather than anything using `pydata_google_auth`?

Reply:
This was mostly to be backwards compatible. I guess people won't be too annoyed if we change the credentials location and they have to reauth?

Another concern is if they use the pydata_google_auth location, it may get confusing if they use that library with different scopes somewhere else.